internal/ksql/command_cluster_configureacls.go

```diff
@@ -24,6 +25,12 @@ func (c *ksqlCommand) newConfigureAclsCommand() *cobra.Command {
 		Args:              cobra.MinimumNArgs(1),
 		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
 		RunE:              c.configureACLs,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Configure ACLs for ksqlDB cluster lksqlc-12345 for topics topic_1 and topic_2:",
+				Code: "confluent ksql cluster configureacls lksqlc-12345 topic_1 topic_2",
```

Comment:
```suggestion
				Text: `Configure ACLs for ksqlDB cluster "lksqlc-12345" for topics "topic-1" and "topic-2":`,
				Code: "confluent ksql cluster configure-acls lksqlc-12345 topic-1 topic-2",
```

Reply:
we're using my_topic in other examples so let's keep that

Reply:
We should definitely standardize this in a later PR, but sounds ok for now!